### PR TITLE
reducing test times for DGM

### PR DESCRIPTION
**Pull request by ayushinav**
Following from the last [comment](https://github.com/SciML/NeuralPDE.jl/pull/802#issuecomment-1976922157) in #802, this reduces the run times for dgm tests.
@sathvikbhagavan @ChrisRackauckas

### CODE REVIEW

File: docs/src/tutorials/dgm.md [hunk 1]
```diff
@@ -84,8 +87,8 @@ xs = sol[x]
 u_MOL = sol[u(t,x)]
 
 # NeuralPDE, using Deep Galerkin Method
-strategy = QuasiRandomTraining(4_000, minibatch= 500);
-discretization= DeepGalerkin(2, 1, 50, 5, tanh, tanh, identity, strategy);
+strategy = QuasiRandomTraining(256, minibatch= 32);
+    discretization= DeepGalerkin(2, 1, 50, 5, tanh, tanh, identity, strategy);
```
Comment (sathvikbhagavan) [inline docs/src/tutorials/dgm.md:91]:
```suggestion
discretization = DeepGalerkin(2, 1, 50, 5, tanh, tanh, identity, strategy)
```

File: docs/src/tutorials/dgm.md [hunk 2]
```diff
@@ -84,8 +87,8 @@ xs = sol[x]
 u_MOL = sol[u(t,x)]
 
 # NeuralPDE, using Deep Galerkin Method
-strategy = QuasiRandomTraining(4_000, minibatch= 500);
-discretization= DeepGalerkin(2, 1, 50, 5, tanh, tanh, identity, strategy);
+strategy = QuasiRandomTraining(256, minibatch= 32);
```
Comment (sathvikbhagavan) [inline docs/src/tutorials/dgm.md:90]:
```suggestion
strategy = QuasiRandomTraining(256, minibatch= 32)
```

File: docs/src/tutorials/dgm.md [hunk 3]
```diff
@@ -84,8 +87,8 @@ xs = sol[x]
...
+    discretization= DeepGalerkin(2, 1, 50, 5, tanh, tanh, identity, strategy);
 @named pde_system = PDESystem(eq, bcs, domains, [t, x], [u(t,x)]);
```
Comment (sathvikbhagavan) [inline docs/src/tutorials/dgm.md:92]:
```suggestion
@named pde_system = PDESystem(eq, bcs, domains, [t, x], [u(t,x)])
```

File: docs/src/tutorials/dgm.md [hunk 4]
```diff
@@ -84,8 +87,8 @@ xs = sol[x]
...
 prob = discretize(pde_system, discretization);
```
Comment (sathvikbhagavan) [inline docs/src/tutorials/dgm.md:93]:
```suggestion
prob = discretize(pde_system, discretization)
```

File: docs/src/tutorials/dgm.md [hunk 5]
```diff
@@ -84,8 +87,8 @@ xs = sol[x]
...
 global iter = 0;
```
Comment (sathvikbhagavan) [inline docs/src/tutorials/dgm.md:94]:
```suggestion
global iter = 0
```

**Review by sathvikbhagavan — COMMENTED** [review]
Left a few small formating changes in the tutorial.
One more thing is the tutorial won't run in the doc build. Use `@example ...` syntax instead of julia markdown.  Ref - https://documenter.juliadocs.org/stable/man/syntax/#@example-block

File: docs/src/tutorials/dgm.md [hunk 6]
```diff
@@ -44,7 +47,7 @@ u(t, 1) & = 0
 ```
 
 ### Copy- Pasteable code
-```julia
+```@example
```
Comment (sathvikbhagavan) [inline docs/src/tutorials/dgm.md:50]:
```suggestion
```@example dgm
```

We can name them such that each example runs in isolation

Comment (ayushinav) [reply docs/src/tutorials/dgm.md:50]: Done!

**Comment by ayushinav** [comment]
Thanks, made the changes. I was curious why the plot did not render as such in the dev docs. I looked at `gpu.md` which used `julia` instead of `@example` but ig it's the only that does so.

**Review by sathvikbhagavan — APPROVED** [review]

**Pull request merged**